task_02.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Creating an appointment system with email blast."""
+
+
+def prepare_email(appointments):
+    """Generates the text of an email to clients.
+
+    Args:
+        appointments: A list of two-item tuples with the client's name and their
+        appointment time as members
+
+    Returns:
+        A new list with the client's email body
+
+    Examples:
+        >>> prepare_email([('Jen', '2015'), ('Max', 'March 3')])
+        ['Dear Jen,\nI look forward to meeting with you on 2015.\nBest,\nMe',
+        'Dear Max,\nI look forward to meeting with you on March 3.\nBest,\nMe']
+        >>> prepare_email([('Mary Moore', '12.1.2015'), ('Jacob Jones', 'May 09,
+        2016')])
+        ['Dear Mary Moore,\nI look forward to meeting with you on 12.1.2015.\n
+        Best,\nMe', 'Dear Jacob Jones,\nI look forward to meeting with you on
+        May 09, 2016.\nBest,\nMe']
+    """
+    emailbody = []
+    for client in appointments:
+        emailstr = 'Dear {},\nI look forward to meeting with you on {}.' \
```

Comment:
You should declare `emailstr` outside of the for loop. The reason is it's unnecessary to keep making that string over and over each time it loops to the next client.